cli/impl/src/main/assembly/helidon.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+main() {
+    local command isDebug
+    findRootDirectory
+    findJavaExecutable
+    buildCommand "$@"
```

Comment:
I know this is inspired from the `mvn.sh` script, but let's make it our own a bit:
- use _ separated lower case method names.
- Use "${}" instead of just "$"

Also, see if you can use https://github.com/koalaman/shellcheck.